src/static/css/pad/layout.css

```
@@ -4,6 +4,14 @@ html, body {
   margin: 0;
   padding: 0;
 }
+
+html.timeslider {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
```

width, margin, and padding are already covered by the above `body, html {}` block:

```suggestion
  height: 100%;
```